src/main/groovy/com/github/jrubygradle/jar/JRubyJarConfigurator.groovy

```diff
@@ -24,15 +24,6 @@ class JRubyJarConfigurator {
         configure()
     }
 
-    @PackageScope
```

Comment:
on second thought: I moved back the shadowJar option for jruby-gradle-jar-plugin BUT things will not really work when you use it since all those embedded jars are gone and jruby-main will look for them and not find them.